ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -103,6 +121,17 @@ azure_connection::read_object(
   const std::string &object_key, int64_t offset, size_t len, void *buf) const
 {
     auto blob_client = _azure_client.GetBlockBlobClient(_object_prefix + object_key);
+
+    // GetProperties will return BlobProperties describing the blob or an exception.
```

Comment:
We could add some detail to this comment to be more useful for the reader:
```
// GetProperties returns a BlobProperties object containing the blob size on success 
// and throws an exception on failure.
```

Reply:
resolved here https://github.com/wiredtiger/wiredtiger/pull/8750/commits/e42f214727eda92b86bcea0d45fc6c00676173e8